content/en/docs/about/community.md

```diff
@@ -5,30 +5,23 @@ weight = 10
 aliases = ["/docs/community/"]
 +++
 
-## Kubeflow Slack
-
-The Official Kubeflow Slack is used for informal discussions among users and contributors.
-
-<a href="https://communityinviter.com/apps/kubeflow/slack">
-  <button class="btn btn-primary py-2 px-5 mb-3">Click to join:<br>Kubeflow Slack</button>
-</a>
-
-| Popular Kubeflow Slack Channels |
-| --- | --- |
-| General Discussion | [#general](https://kubeflow.slack.com/archives/C7REE0ETX)
-| Feature Requests | [#feature-requests](https://kubeflow.slack.com/archives/C01A7RYEYMB)
-| Job Postings | [#job-postings](https://kubeflow.slack.com/archives/CJ9PJE5FS)
-| Kubeflow - Pipelines | [#kubeflow-pipelines](https://kubeflow.slack.com/archives/CE10KS9M4)
-| Kubeflow - Notebooks | [#kubeflow-notebooks](https://kubeflow.slack.com/archives/CESP7FCQ7)
-| Kubeflow - Training Operator | [#kubeflow-training-operator](https://kubeflow.slack.com/archives/C985VJN9F)
-| Kubeflow - Katib | [#kubeflow-katib](https://kubeflow.slack.com/archives/CE0BURK1B)
-| KServe | [#kserve](https://kubeflow.slack.com/archives/CH6E58LNP)
-| Platform - AWS | [#platform-aws](https://kubeflow.slack.com/archives/CKBA5D0MU)
-| Platform - Azure | [#platform-azure](https://kubeflow.slack.com/archives/CUW6SLCPR)
-| Platform - GCP | [#platform-gcp](https://kubeflow.slack.com/archives/CKH7V1M7F)
-| Users - China | [#users-china](https://kubeflow.slack.com/archives/C93HYNM9C)
-| Users - Korea | [#users-korea](https://kubeflow.slack.com/archives/CKPCJB9AP)
-| Users - Oceania | [#users-oceania](https://kubeflow.slack.com/archives/C023ZN1R9FC)
+## Kubeflow Slack Channels
+
+Kubeflow is part of the vibrant CNCF community. You are welcome to create your CNCF Slack account [here](https://slack.cncf.io/) and then join the following list of Kubeflow Slack channels.
+
+| Description                 | Link                                                                             |
+|-----------------------------|----------------------------------------------------------------------------------|
+| Announcements               | [#kubeflow-announcements](https://app.slack.com/client/T08PSQ7BQ/C01EV0FV154)    |
+| GSoC Mentors                | [#kubeflow-gsoc-mentors](https://app.slack.com/client/T08PSQ7BQ/C06LJP14PB8)     |
```

Comment:
Do we need to add this channel here ?

Reply:
I am ok to make this channel private. What do you think @rareddy ?

Reply:
This channel is already private, and MUST stay private, its for discussing sensitive topics that students should not see.

Reply:
It's currently public.

Reply:
removed. Will make it private once more mentors join